stdlib/universe/hourSelection_test.go

```diff
@@ -0,0 +1,424 @@
+				},
+			},
+		},
+		{
+			name: "no group key -- across multiple days",
+			spec: &universe.HourSelectionProcedureSpec{Start: 2, Stop: 8},
+			data: []flux.Table{
+				&executetest.Table{
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "tag0", Type: flux.TString},
+						{Label: "tag1", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{execute.Time(3600000000000), "a", "d"},
+						{execute.Time(3600000000000 * 2), "a", "b"},
+						{execute.Time(3600000000000 * 4), "a", "c"},
+						{execute.Time(3600000000000 * 8), "a", "e"},
+						{execute.Time(3600000000000 * 26), "a", "f"},
+						{execute.Time(3600000000000 * 28), "a", "g"},
+						{execute.Time(3600000000000 * 32), "a", "h"},
+					},
+				},
+			},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TString},
+					{Label: "tag1", Type: flux.TString},
+				},
+				Data: [][]interface{}{
+					{execute.Time(3600000000000 * 2), "a", "b"},
+					{execute.Time(3600000000000 * 4), "a", "c"},
+					{execute.Time(3600000000000 * 8), "a", "e"},
+					{execute.Time(3600000000000 * 26), "a", "f"},
+					{execute.Time(3600000000000 * 28), "a", "g"},
+					{execute.Time(3600000000000 * 32), "a", "h"},
+				},
+			}},
+		},
+		{
+			name: "no group key -- nil",
+			spec: &universe.HourSelectionProcedureSpec{Start: 8, Stop: 24},
+			data: []flux.Table{
+				&executetest.Table{
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "tag0", Type: flux.TString},
+						{Label: "tag1", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{execute.Time(3600000000000), "a", "b"},
+						{execute.Time(3600000000000 * 3), "a", "c"},
+						{execute.Time(3600000000000 * 5), "a", "b"},
+						{execute.Time(3600000000000 * 7), "a", "d"},
+					},
+				},
+			},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TString},
+					{Label: "tag1", Type: flux.TString},
+				},
+				Data: [][]interface{}(nil),
+			}},
+		},
+		{
+			name: "no group key -- outside range",
+			spec: &universe.HourSelectionProcedureSpec{Start: 25, Stop: 40},
+			data: []flux.Table{
+				&executetest.Table{
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "tag0", Type: flux.TString},
+						{Label: "tag1", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{execute.Time(3600000000000 * 6), "a", "b"},
+						{execute.Time(3600000000000 * 25), "a", "c"},
+						{execute.Time(3600000000000 * 33), "a", "b"},
+						{execute.Time(3600000000000 * 39), "a", "d"},
+					},
+				},
+			},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TString},
+					{Label: "tag1", Type: flux.TString},
+				},
+				Data: [][]interface{}(nil),
```

Comment:
mmmmm this is curious.

Wouldn't it be better to error if one provide `start` or `stop` `> 23` or `< 0`. I wouldn't expect  this result given the intrinsic periodicity of time....

Otherwise, you could say that `start` and `stop` are implicitly `%24` and this test  would be the same as:

```
&universe.HourSelectionProcedureSpec{Start: 1, Stop: 16},
```